### PR TITLE
Agreement

### LINKED ISSUE #38: Add TOS/privacy statement pages
The site should explicitly list terms of service and privacy, somewhere.

### LINKED ISSUE #53: Confirm users are 13 years old
Have a checkmark when registering that has users confirm they are at least 13 years old.

### PR DESCRIPTION
**Pull request by rebbrunner**
Fixes #38 and fixes #53
- Add privacy agreement
- Add 13+ confirmation

### CODE REVIEW

**Review by psybers — CHANGES_REQUESTED** [review]
These policies need to be actual pages.  Each policy should be in its own template file (so that you can grab the contents and display in the modal).

**Comment by psybers** [comment]
Maybe I was a bit unclear here, I apologize. The privacy policy should be 1 file.  The service agreement should be 1 file (not 16 files).

**Comment by psybers** [comment]
Is it possible to get the text up next to the checkboxes? Right now that whole 'row' is clickable, even the white space.  Yet the text is not clickable because it is 'help' text.

Screen Shot 2019-04-11 at 2 24 51 PM

**Comment by psybers** [comment]
Regarding the modal that pops up.  Can we make it much wider?  I think theres a way to make modals 'wide' instead of this skinny thing we get now.  Also the text - can we stick that into something that scrolls?  So ideally the dialog is fully visible (including the button to close it) and the text scrolls within the modal.

File: website/static/website/style.css [hunk 1]
```diff
@@ -64,3 +64,18 @@ body {
...
+    color: inherit;
+    border: none; 
+    padding: 0 !important;
+    font: inherit;
+    border-bottom: 1px solid #444; 
+    cursor: pointer;
+}
+
+.modal-body {
+    max-height: calc(100vh - 212px);
```
Comment (rebbrunner) [inline website/static/website/style.css]: 212px = height of header and footer together

Comment (psybers) [reply website/static/website/style.css]: No. The website scales.  If you zoom in, the footer will grow in height because the contents wrap.

**Review by psybers — APPROVED** [review]

**Pull request merged**